utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -438,6 +440,7 @@
 "The Apache Software Licence, Version 2.0": Apache-2.0
 "The Apache Software License, Version 1.1": Apache-1.1
 "The Apache Software License, Version 2.0": Apache-2.0
+"The BSD License": BSD-3-Clause
```

Comment:
Note that this mapping actually is ambiguous (it could be the 3-clause or the 2-clause variant), and we aim to not add any more ambiguous mapping. Also see the discussion at https://github.com/oss-review-toolkit/ort/pull/8139.